pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1729,3 +1728,18 @@ func getActualWildcardAndLocalHost(node *model.Proxy) (string, string) {
 	}
 	return WildcardIPv6Address, LocalhostIPv6Address
 }
+
+// getSidecarInboundBindIp returns the IP that the proxy can bind to along with the sidecar specified port.
+// It looks for an unicast address, if none found, then the default wildcard address is used.
+// This will make the inbound listener bind to instance_ip:port instead of 0.0.0.0:port where applicable.
+func getSidecarInboundBindIp(node *model.Proxy) string {
```

Comment:
@shakti-das , sorry for the picky linter: Variable Naming Convention:
Variable name is generally using Camel Case style, but when you have unique nouns, should apply following rules:

- If the variable is private, and the unique noun is the first word, then use lower cases, e.g. apiClient.
- Otherwise, use the original cases for the unique noun, e.g. APIClient, repoID, UserID.

 "IP" is commonly identified as a unique noun ;-)

Please, address this comment as it will block circleCI tests.

Reply:
Done. Thanks.